package.json

```diff
@@ -50,6 +61,8 @@
     "lodash.throttle": "^4.1.1",
     "request": "^2.67.0",
     "request-promise": "^4.2.0",
+    "socks-proxy-agent": "^3.0.1",
+    "thunkify": "^2.1.2",
```

Comment:
- `thunkify ` can be removed
- `socks-proxy-agent` is already a dependency in `devDependencies` at line 41 - one of them has to get removed